v2/pkg/reporting/exporters/json-exporter/json-exporter.go

```diff
@@ -0,0 +1,63 @@
+package json_exporter
```

Comment:
we can rename this pkg to `json` instead of `json_exporter` as it is already under `exporters` directory

Reply:
Resolved in [`d35230b`](https://github.com/projectdiscovery/nuclei/pull/3466/commits/d35230bca77d349d574f221272ce1e52390fd673).

Also renamed the configuration field option `"json"` from `"json-exporter"` to match the others in the same object (`"sarif"`, `"markdown"`, etc)